### PR TITLE
Update zh.js FINAL VERSION (reviewed by WpyGG) (v3.8.1)

### PR DESCRIPTION
**Pull request by UnityEthan**
This is based on the last change i made, but i added more translations, and improved some of the existing ones. No Google translation is involved. DM Stonkman#0948 on discord for details

### CODE REVIEW

File: zh.js
Comment (mty22) [inline zh.js:617]: just need to fix whitespace on end of line

Comment (UnityEthan) [reply zh.js:617]: umm idk what spaces do u mean but ill look into it

Comment (UnityEthan) [reply zh.js:617]: oh, i see what you mean. i just cant edit anymore

**Comment by mty22** [comment]
still work in progress

**Comment by WpyGG4125** [comment]
@mty22 what do I do if I have completed the last part of the translation and merged both files?

**Comment by mty22** [comment]
I've made comments in Discord, please have a look:

https://discord.com/channels/448194623580667916/831383180086607922/834227613236461588
https://discord.com/channels/448194623580667916/831383180086607922/834409630837768222

File: zh.js
Comment (mty22) [inline zh.js:571]: Just noticed that you've used: `。` here, and other parts of the language file has `.`

Can you please update these?

File: zh.js
Comment (mty22) [inline zh.js:720]: Same case here as with my previous comment about: `。`

File: zh.js
Comment (mty22) [inline zh.js:732]: Same case here as with my previous comment about: `。`

File: zh.js
Comment (mty22) [inline zh.js:736]: Same case here as with my previous comment about: `。`

**Review by mty22 — CHANGES_REQUESTED** [review]
Just a few minor changes.

Also @WpyGG4125 can you please confirm that you've reviewed? :)

**Comment by WpyGG4125** [comment]
Yes I confirm that I have reviewed and most parts are correct 👍

There are just some little errors:
dev.header is missing, I would suggest to name it 开发者
clans.create is having an excessive 工 (IMO 公会 could be better than 战队 but it's fine)
and I would suggest changing *Premium* to 会员 instead of leaving it as English

Besides these, the file is perfect and ready to go

@mty22 @UnityEthan

**Comment by mty22** [comment]
@UnityEthan can you update the pull request as per @WpyGG4125 's suggestions? We're good to merge after that.

**Comment by mty22** [comment]
@UnityEthan please tell me the work breakdowns between all users who contributed.

File: zh.js
Comment (mty22) [inline zh.js:305]: `(` and `（` are different, are you able to fix this up please?

Ideally go with `(` and `)` to help reduce whitespace

Comment (mty22) [reply zh.js:305]: can you fix this one through out the whole file? find and replace should do the trick!

**Review by mty22 — CHANGES_REQUESTED** [review]

**Comment by UnityEthan** [comment]
**EVERYTHING BELOW IS ASSUMING THAT THE REWARD ON TRADITIONAL IS 50k:**
Credits: 1) Stonkman#0948, ign: GV_EthanW, contributed to 80% of the translation
              2) WpyGG#4125, ign: WpyGG, contributed to 10% of the translation
              3) Awesome_Boy#7555, ign: Awesome_Boy, contributed to 10% of the translation

**Comment by mty22** [comment]
> **EVERYTHING BELOW IS ASSUMING THAT THE REWARD ON TRADITIONAL IS 50k:**
> Credits: 1) Stonkman#948, ign: GV_EthanW, contributed to 80% of the translation
> 2) WpyGG#4125, ign: WpyGG, contributed to 10% of the translation
> 3) Awesome_Boy#7555, ign: Awesome_Boy, contributed to 10% of the translation

this is the simplified chinese translation, not traditional.

**Comment by mty22** [comment]
need final review since last changes: https://github.com/mty22/krunker.io-translations/pull/104/files/6b14f344502acef23a7a54d1f3fcc4c31c8414a7..bc70591e7d1f6914fde719cb64649c1f03799ff4

reviewer can be anyone except UnityEthan.

also @UnityEthan please provide work breakdowns for **Simplified Chinese** update, along with discord users + ign's.

**Comment by mty22** [comment]
need final review since last changes: https://github.com/mty22/krunker.io-translations/pull/104/files/6b14f344502acef23a7a54d1f3fcc4c31c8414a7..bc70591e7d1f6914fde719cb64649c1f03799ff4

reviewer can be anyone except UnityEthan.

also @UnityEthan please provide work breakdowns for **Simplified Chinese** update, along with discord users + ign's.

File: zh.js
Comment (TridentYT) [inline zh.js:486]: 粘贴地图数据

**Comment by UnityEthan** [comment]
EVERYTHING BELOW IS ASSUMING THAT THE REWARD ON TRADITIONAL IS 50k:
Credits: 1) Stonkman#948, ign: GV_EthanW, contributed to 80% of the translation
2) WpyGG#4125, ign: WpyGG, contributed to 10% of the translation
3) Awesome_Boy#7555, ign: Awesome_Boy, contributed to 10% of the translation
**NOTE: THE PERCENTAGES SOLELY REPRESENT THE AMOUNT OF WORK BEING PUT INTO THIS PARTICULAR TRANSLATION, THEY CAN SORT TRADITIONAL OUT THEMSELVES**

**Comment by mty22** [comment]
> EVERYTHING BELOW IS ASSUMING THAT THE REWARD ON TRADITIONAL IS 50k

@UnityEthan you're clearly not understanding something here.

This is the Simplified Chinese translation, **not** the Traditional Chinese translation.

The question I've asked the workload breakdown for the Simplified Chinese translation.

**Comment by WpyGG4125** [comment]
> > EVERYTHING BELOW IS ASSUMING THAT THE REWARD ON TRADITIONAL IS 50k
> 
> @UnityEthan you're clearly not understanding something here.
> 
> This is the Simplified Chinese translation, **not** the Traditional Chinese translation.
> 
> The question I've asked the workload breakdown for the Simplified Chinese translation.

@mty22 We had some discussions about the division of rewards so we decided to change the rate depending on the reward amount of traditional chinese, if tr chinese offers less rewards then the contribution rate might change

**Comment by mty22** [comment]
> > > EVERYTHING BELOW IS ASSUMING THAT THE REWARD ON TRADITIONAL IS 50k
> > 
> > 
> > @UnityEthan you're clearly not understanding something here.
> > This is the Simplified Chinese translation, **not** the Traditional Chinese translation.
> > The question I've asked the workload breakdown for the Simplified Chinese translation.
> 
> @mty22 We had some discussions about the division of rewards so we decided to change the rate depending on the reward amount of traditional chinese, if tr chinese offers less rewards then the contribution rate might change

In context, Traditional Chinese is a "new language" to Krunker, where as Simplified Chinese is not.

The KR payout for this pull request has zero relation to Traditional Chinese translation as that's completely different.

The KR rewards are set per pull request as I've [already noted here](https://github.com/mty22/krunker.io-translations/blob/main/kr_payment_guide.txt). This won't change unless there's some extremely rare circumstance which dictates once-off change (that of which would be assessed on a case-by-case basis).

Please, stop mixing them together. Keep it simple. This is a public forum, and rules must be adhered to.

Again, please let me know the work breakdowns **_specifically_** for Simplified Chinese.

**Comment by UnityEthan** [comment]
Contributors: 1) Stonkman#094, 90%, ign: GV_EthanW
                      2) WpyGG#4125, 10%, ign: WpyGG

**Comment by WpyGG4125** [comment]
> Contributors: 1) Stonkman#94, 90%, ign: GV_EthanW
> 2) WpyGG#4125, 10%, ign: WpyGG

I have 15% contribution 😬

**Comment by Awesome-Boy-001** [comment]
> Contributors: 1) Stonkman#94, 90%, ign: GV_EthanW
> 2) WpyGG#4125, 10%, ign: WpyGG

I've also contributed to 10% of the translation

**Comment by WpyGG4125** [comment]
Let's just stick to the original division
1) Stonkman#948, ign: GV_EthanW, contributed to 80% of the translation
2) WpyGG#4125, ign: WpyGG, contributed to 10% of the translation
3) Awesome_Boy#7555, ign: Awesome_Boy, contributed to 10% of the translation
@UnityEthan @mty22 @Awesome-Boy-001

File: zh.js
Comment (mty22) [inline zh.js:305]: gotta fix this

File: zh.js
Comment (mty22) [inline zh.js:306]: gotta fix this for consistency sake

File: zh.js
Comment (mty22) [inline zh.js:425]: https://github.com/mty22/krunker.io-translations/blob/main/en.js#L425

need to check what is set across all files

**Comment by mty22** [comment]
> Let's just stick to the original division
> 
> 1. Stonkman#948, ign: GV_EthanW, contributed to 80% of the translation
> 2. WpyGG#4125, ign: WpyGG, contributed to 10% of the translation
> 3. Awesome_Boy#7555, ign: Awesome_Boy, contributed to 10% of the translation


we will go with this

**Comment by mty22** [comment]
OK, merge conflict fixed.

**Comment by mty22** [comment]
KB roles sorted, KR queued.

**Pull request merged**
